code/modules/paperwork/paperbin.dm

```diff
@@ -130,23 +130,21 @@
 
 /obj/item/paper_bin/examine(mob/user)
 	. = ..()
+	var/total_paper = LAZYLEN(papers)
```

Comment:
this doesn't seem relevant to this PR?

Reply:
one would think
I setup code that calls update_overlays on init, for reasons, and since paperbins used the var this replaced as both a cache of the paper count AND a way to tell themselves on init how much paper to spawn, it broke
Fuck I forgot to mention that in the pr. I will.

https://github.com/tgstation/tgstation/pull/67902#issuecomment-1191952086

Reply:
https://github.com/LemonInTheDark/tgstation/commit/644813af784dc9b5a900e2d4d3f6dfad176fc54f